spec/rubocop/cop/rspec/variable_definition_spec.rb

```diff
@@ -11,17 +11,15 @@
       RUBY
     end
 
-    it 'registers an offense for interpolated string' do
-      expect_offense(<<~'RUBY')
+    it 'does not register offense for interpolated string' do
+      expect_no_offenses(<<~'RUBY')
         let("user-#{id}") { 'Adam' }
-            ^^^^^^^^^^^^ Use symbols for variable names.
       RUBY
     end
 
-    it 'registers an offense for multiline string' do
-      expect_offense(<<~'RUBY')
-        let("user"\
-            ^^^^^^^ Use symbols for variable names.
+    it 'does not register offense for multiline string' do
+      expect_no_offenses(<<~'RUBY')
+        let("user" \
             "-foo") { 'Adam' }
```

Comment:
Doesn't fall into an interpolated string category, but I understand that they are both `dstr_type?`, pretty similar AST-wise and innecesarily hard to distinguish:
```
$ ruby-parse -e '"a" \
quote> "b"'
(dstr
  (str "a")
  (str "b"))
```
```
$ ruby-parse -e '"a#{"b"}"'
(dstr
  (str "a")
  (begin
    (str "b")))
```

Reply:
And we shouldn’t recommend converting a multi-line string into a multi-line symbol. Right?

Reply:
Indeed!